src/explore-education-statistics-admin/src/pages/admin-dashboard/components/ReleaseSummary.tsx

```diff
@@ -40,21 +38,17 @@ const ReleaseSummary = ({
       summary={getReleaseSummaryLabel(release)}
       summaryAfter={
         <TagGroup className="govuk-!-margin-left-2">
-          <Tag>{getReleaseStatusLabel(release.status)}</Tag>
-
-          {release.amendment && <Tag>Amendment</Tag>}
-
+          {release.status !== 'Approved' && (
+            <Tag>{getReleaseStatusLabel(release.status)}</Tag>
+          )}
           {release.status === 'Approved' && (
-            <LazyLoad
```

Comment:
With the lazy loading is removed, I think we're now requesting the status of every approved release on the dashboard as soon as the dashboard loads? I think the lazy loading was added to prevent this from happening, so we should keep it.

Reply:
Seems it is requesting the status for every approved release under the topic currently selected - not sure if we should consider that an issue or not.

Reply:
@markhiveit I've restored the lazy loading now. Had a look at stopping the endpoint being called twice, think it'll mean a bit of refactoring of the components so probably out of scope for this PR.